bridge_adaptivity/module/templates/module/modals/delete_collection_warning.html

```diff
@@ -0,0 +1,29 @@
+{% load bootstrap3 %}
```

Comment:
This template is the same as previous one except for two lines of the text why don't use one with the appropriate statement?